### PR TITLE
UNI-306: Custom X and Y axes. Anomaly in legend.

### PR DESCRIPTION
**Pull request by brev**
UNI-306: Custom X and Y axes. Anomaly in legend.

https://jira.numenta.com/browse/UNI-306

@lscheinkman @marionleborgne please review anytime.

### CODE REVIEW

**Comment by lscheinkman** [comment]
The range anomalies moved to the top of the chart.
anomaly

**Comment by lscheinkman** [comment]
Anomaly label always showing **3%**
wrong label

**Comment by brev** [comment]
@lscheinkman @marionleborgne fixes are in here, please review again. thanks.

File: unicorn/js/browser/components/ModelData.jsx [hunk 1]
```diff
@@ -168,12 +169,12 @@ export default class ModelData extends React.Component {
 
     if (
       modelData &&
-      modelData[column] &&
-      modelData[column][DATA_INDEX_ANOMALY]
+      modelData[row] &&
+      modelData[row][DATA_INDEX_ANOMALY]
     ) {
-      anomaly = modelData[column][DATA_INDEX_ANOMALY];
+      anomaly = anomalyScale(modelData[row][DATA_INDEX_ANOMALY]);
```
Comment (lscheinkman) [inline unicorn/js/browser/components/ModelData.jsx]: @brev `anomalyScale` is not really the appropriate value for the label. It is just used for scaling the anomaly bar height. `10%` for `green`, `20%` for `yellow` and `30%` for `red`.
You should use the `anomaly` value as is for the label

**Comment by marionleborgne** [comment]
wow this looks very nice now @brev. 

LGTM :+1:

File: unicorn/js/browser/components/ModelData.jsx (hunk 1 above)
Comment (lscheinkman) [inline unicorn/js/browser/components/ModelData.jsx]: Anomaly value should be **100%** not **30%**
anomaly value

**Comment by brev** [comment]
Removed anomalyScale() call from text value display

**Comment by lscheinkman** [comment]
:+1: LGTM

**Comment by marionleborgne** [comment]
:metal:

**Pull request merged**